pkg/api/server/register_containers.go

```diff
@@ -107,15 +124,19 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//      "$ref": "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}"), APIHandler(s.Context, generic.RemoveContainer)).Methods(http.MethodDelete)
 	// swagger:operation GET /containers/{nameOrID}/json containers getContainer
-	//
-	// Inspect Container
-	//
 	// ---
+	// summary: Inspect container
+	// description: Return low-level information about a container.
 	// parameters:
 	//  - in: path
-	//    name: nameOrID
+	//    name: nameorid
```

Comment:
Other endpoints use the camel-case while the [docker docs](https://docs.docker.com/engine/api/v1.40/#operation/ContainerDelete) name it `id`. Does the identifier impact if the argument matches?